examples/js/postprocessing/MaskPass.js

```diff
@@ -25,11 +25,12 @@ THREE.MaskPass.prototype = {
 	render: function ( renderer, writeBuffer, readBuffer, delta, maskActive ) {
 
 		var context = renderer.context;
+		var state = renderer.state;
 
 		// don't update color or depth
 
-		context.colorMask( false, false, false, false );
-		context.depthMask( false );
+		state.setColorWrite( false );
+		state.setDepthWrite( false );
```

Comment:
I think the problem is caused by these lines. Setting the the WebGL state for color and depth buffer via `WebGLState` is actually useless, because `clolorWrite` and `depthWrite` properties of the material seems to overwriting these method calls.

Have a look at this new [fiddle](http://jsfiddle.net/q73ok880/2/). I added two lines below your mask material to show the effect.

```javascript
// mask scene
var geometry = new THREE.PlaneBufferGeometry( 40, 40 );
var material = new THREE.MeshBasicMaterial( { color: 0xffff00 } ); // bright yellow to see it - it shouldn't show
material.colorWrite = false;
material.depthWrite = false;
```
Um, looks like we can't use `state.setColorWrite` and `state.setDepthWrite` in `MaskPass` right now. Do you think we can overwrite the material properties in any way? I mean force something like a "global state" that has a higher priority than material properties?